.travis/test_06_script.sh

```diff
@@ -65,3 +69,56 @@ if [ "$RUN_FUNCTIONAL_TESTS" = "true" ]; then
   DOCKER_EXEC test/functional/test_runner.py --combinedlogslen=4000 --coverage --quiet --failfast ${extended}
   END_FOLD
 fi
+
+# macOS build - does not use docker
+elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
+
+BEGIN_FOLD autogen
+./autogen.sh || (EXIT_CODE=$?; echo "./autogen.sh exited with $EXIT_CODE"; exit $EXIT_CODE)
+echo "mkdir build"
+mkdir build || (echo "could not create build directory"; exit 1)
+echo "cd build"
+cd build || exit 1
+END_FOLD
+
+echo "configure"
+BEGIN_FOLD configure
+../configure --cache-file=config.cache $BITCOIN_CONFIG_ALL $BITCOIN_CONFIG || ( cat config.log && false)
```

Comment:
Double quote to prevent globbing and word splitting.

Reply:
I think the config arguments should be split off. No?

Reply:
Oh, sorry. Was a bit too quick with that comment :-)